packages/react-preferences/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@trycourier/react-preferences",
+  "version": "0.0.1-dev",
```

Comment:
should match the other packages

Reply:
We want to match the exact version? or you are suggesting to use `-alpha` suffix?